ratings/app.py

```diff
@@ -0,0 +1,9 @@
+from flask import Flask, jsonify
+from pymongo import MongoClient
+from bson import ObjectId
+
+# Connect to MongoDB
+client = MongoClient('localhost', 27017)
+db = client['restaurant']  # Replace 'restaurant' with your database name
+ratings = db['rating']
```

Comment:
line 8, i am not sure what the purpose is

Reply:
Well I was testing a way to point to the ratings property in the DB under the restaurant category. This will allow me to later modify those those ratings for certain restaurants in my AddRatings method. (etc. the file is still not complete so its still in a testing phase)

Reply:
gotcha, something like this will accomplish the same thing if it helps in the future 
users = list(db['rating'].find({}))